src/main/java/mate/academy/controller/BookController.java

```diff
@@ -27,8 +29,8 @@ public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public Page<BookDto> getAll(Pageable pageable) {
```

Comment:
```suggestion
    public Page<BookDto> getAll(@ParameterObject @PageableDefault Pageable pageable) {
```
public List<BookDto> is also good approach